styles/summertime/themes.s2

```diff
@@ -1377,6 +1378,104 @@ function Page::print_theme_stylesheet() { """
 """; }
 
 
+<<<<<<< HEAD
```

Comment:
Missed this! Thought that the merge being resolved meant that this had been resolved. Going to back out this commit. @momijizukamori please try again without these two lines :)